tests/base.conf.js

```diff
@@ -22,6 +23,10 @@ class BaseConfig {
           args: chromeArgs
         }
       },
+      jasmineNodeOpts: {
+        // makes default jasmine reporter not display dots for every spec
+        print: () => {}
+      },
```

Comment:
You don't like the dots?

Occasionally it's useful to know that the tests are still running.

Can we print the name of the test that's being executed instead of a dot? Potentially we could only print the name if in debug mode?

Reply:
I don't like the dots, the dots along with the names look stange - the default reporter displays a `.` for a successful test and an `F` for a failure. 
Together they look like this:
![image](https://user-images.githubusercontent.com/35681649/58343468-bb3d6f00-7e5b-11e9-81df-cd2a18a940ac.png)

I do like the names, and I'd like to see them in my Travis builds. 
The reporter offers plenty of customization. Would you rather the name of the test be displayed at start and then a separate message upon success/failure?

Reply:
No that's perfect, without the dots! It's fairly easy to work out which test is running if you can see which test has just completed so leave it as is.